pages/illustration.html

```diff
@@ -0,0 +1,93 @@
+        </li>
+        <li class="header-social-btn">
+          <a href="https://www.linkedin.com/in/june-valentino/" target="_blank"><?xml version="1.0"?><svg fill="#000000" xmlns="http://www.w3.org/2000/svg"  viewBox="0 0 24 24">    <path d="M19,3H5C3.895,3,3,3.895,3,5v14c0,1.105,0.895,2,2,2h14c1.105,0,2-0.895,2-2V5C21,3.895,20.105,3,19,3z M9,17H6.477v-7H9 V17z M7.694,8.717c-0.771,0-1.286-0.514-1.286-1.2s0.514-1.2,1.371-1.2c0.771,0,1.286,0.514,1.286,1.2S8.551,8.717,7.694,8.717z M18,17h-2.442v-3.826c0-1.058-0.651-1.302-0.895-1.302s-1.058,0.163-1.058,1.302c0,0.163,0,3.826,0,3.826h-2.523v-7h2.523v0.977 C13.93,10.407,14.581,10,15.802,10C17.023,10,18,10.977,18,13.174V17z"/></svg></a>
+        </li>
+        <li class="header-social-btn">
+          <a href="https://twitter.com/vintydraws" target="_blank"><?xml version="1.0"?><svg fill="#000000" xmlns="http://www.w3.org/2000/svg"  viewBox="0 0 24 24">  <path d="M22,3.999c-0.78,0.463-2.345,1.094-3.265,1.276c-0.027,0.007-0.049,0.016-0.075,0.023c-0.813-0.802-1.927-1.299-3.16-1.299 c-2.485,0-4.5,2.015-4.5,4.5c0,0.131-0.011,0.372,0,0.5c-3.353,0-5.905-1.756-7.735-4c-0.199,0.5-0.286,1.29-0.286,2.032 c0,1.401,1.095,2.777,2.8,3.63c-0.314,0.081-0.66,0.139-1.02,0.139c-0.581,0-1.196-0.153-1.759-0.617c0,0.017,0,0.033,0,0.051 c0,1.958,2.078,3.291,3.926,3.662c-0.375,0.221-1.131,0.243-1.5,0.243c-0.26,0-1.18-0.119-1.426-0.165 c0.514,1.605,2.368,2.507,4.135,2.539c-1.382,1.084-2.341,1.486-5.171,1.486H2C3.788,19.145,6.065,20,8.347,20 C15.777,20,20,14.337,20,8.999c0-0.086-0.002-0.266-0.005-0.447C19.995,8.534,20,8.517,20,8.499c0-0.027-0.008-0.053-0.008-0.08 c-0.003-0.136-0.006-0.263-0.009-0.329c0.79-0.57,1.475-1.281,2.017-2.091c-0.725,0.322-1.503,0.538-2.32,0.636 C20.514,6.135,21.699,4.943,22,3.999z"/></svg></a>
+        </li>
+        <li class="header-social-btn">
+          <a href="mailto:vintydesign@gmail.com" target="_blank"><?xml version="1.0" encoding="UTF-8" standalone="no"?> <svg viewBox="0 0 24 24" version="1.1"> <path d="M 12,3.6666667 C 16.595,3.6666667 20.333333,7.405 20.333333,12 20.333333,16.595 16.595,20.333333 12,20.333333 7.405,20.333333 3.6666667,16.595 3.6666667,12 3.6666667,7.405 7.405,3.6666667 12,3.6666667 Z M 12,2 C 6.4775,2 2,6.4775 2,12 2,17.5225 6.4775,22 12,22 17.5225,22 22,17.5225 22,12 22,6.4775 17.5225,2 12,2 Z M 12,12.458333 7.0066667,8.65 H 16.9925 Z M 12,13.531667 7,9.6741667 V 15.316667 H 17 V 9.6741667 Z" style="stroke-width:0.833333" /> </svg></a>
+        </li>
+      </ul>
+    </header>
+    <nav class="nav">
+      <!-- navigation -->
+      <ul class="nav-btn">
+        <li>
+          <a href="software.html">software</a>
+        </li>
+        <li>
+            <a href="illustration.html" class="active-section">illustration</a>
+        </li>
+        <li>
+          <a href="about.html">about</a>
+        </li>
+      </ul>
+    </nav>
+    <main class="main-section">
+      <!-- page content -->
+      <section class="illustration-contents">
```

Comment:
I would recommend only using the `<main>` tag here since there is only one `<section>` inside `<main>`.